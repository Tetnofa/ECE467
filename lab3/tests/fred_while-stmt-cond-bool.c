int f1() {
  int a;
  bool b;
  while (1) {
  }
  while (true) {
  }
  while (true) {
  }
  while (a + a) {
  }
  while (a == a && b) {
  }
}
