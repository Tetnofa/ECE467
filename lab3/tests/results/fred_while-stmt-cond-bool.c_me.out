sema: 4:2 : invalid condition in while statement
sema: 10:2 : invalid condition in while statement
